src/chapters/aspdotnet-model-validation/validation-attributes.rst

```diff
@@ -0,0 +1,184 @@
+Validation Attributes
+=====================
+
+Within the model of a C# web application, we can define validation rules using attributes.
```

Comment:
Suggested edit: "Within the model or ViewModel of a ..."

Students may not be thinking of a ViewModel as a special type of model class, and ViewModels are where most validation will take place.

Reply:
Done!